src/main/java/org/fundacionjala/coding/cesar/FizzBuzz.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.cesar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class FizzBuzz {
+    private static final int FIVE = 5;
+    private static final int THREE = 3;
+    /**
+     * @param range List from int.
+     * @return list String replace.
+     */
+    public List<String> doBuzzFizz(final List<Integer> range) {
+
+        List<String> lst = range
+                .stream()
+                .map(i -> Integer.toString(i))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

Reply:
You can split this method in two
one of them given one number will return fizz, buzz, or fizz buzz